### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by iamsahilsonawane** [flutter-form-builder-ecosystem/phone_number#84] [General]
@deandreamatias We need to update the change log now, with all the updates, and release the next version on pub dev. Also, it would be good if we could add an updated change log checkbox in the pull request template.

**Comment by iamsahilsonawane** [comment]
Also, I'll now be working finally on the web support for the plugin.

**Comment by deandreamatias** [comment]
To change pull request template, we need change [this file](https://github.com/flutter-form-builder-ecosystem/.github/blob/main/PULL_REQUEST_TEMPLATE.md) but be careful because this template is apply for all packages in ecosystem.

About release, I can do it. Will be the `0.13.0` version right?

**Reply by iamsahilsonawane** [reply]
Yes, that would be perfect.

**Reply by iamsahilsonawane** [reply]
About the pull request template, I don't have access to the notion file.

We just need something like: 
- [ ] I have updated the CHANGELOG and increased the version

I think there should be a note on how to update the version number. 
Like for e.g.
1. If there's a change in non-functional components. Like change in docs, or refactors. - there shouldn't be a new version number, just adding a new point in CHANGELOG in previous version will work fine.
2. If there's a breaking change, the version should be updated to N.X.X for e.g.
3. If there's a normal change, depending on the change, X.N.N should be updated.

These are just for examples for what I think. There may be a better way too, do suggest what you think about this.

**Reply by deandreamatias** [reply]
Sorry for link. I updated now.

About increased version, I don't like that every pull request increase version. Only who make a release that can update version.
About changelog, I want use [this solution](https://github.com/flutter-form-builder-ecosystem/phone_number/blob/0bf0a6fcc089a924a637dabe71d1c65e115bef11/.github/workflows/base.yaml#L80) to autogenerate changelog but in this moment this don't work :(
Maybe need extra configuration

**Reply by iamsahilsonawane** [reply]
Ohh that's good! Actually, I contributed to the flutter/plugins repo and they use this kind of system there. But no issues, I like what you're up to here.
